Bottom: 4da5b80cf646f7b0237b8592242439a7ff9308b9
Top:    b6ef4a399a38af14bab4802c0a24e2dbd95e2ba7
Author: Pavel Roskin <proski@gnu.org>
Date:   2014-12-23 22:20:52 -0500

Use -p1 flag for patch

It's needed for any patch affecting more than the top-level directory.

---

diff --git a/farm-patch.in b/farm-patch.in
index 07cc1cd..e46afe5 100644
--- a/farm-patch.in
+++ b/farm-patch.in
@@ -16,8 +16,8 @@ for srctree in $(ls -d $farmsrc/* | sort -V); do
 	git reset -q --hard
 	git clean -q -x -d -f
 	for patch in $confdir/kernel-patches/*.patch; do
-		if patch --dry -s --forward -i $patch >/dev/null; then
-			patch -s --forward -i $patch
+		if patch --dry -s -p1 --forward -i $patch >/dev/null; then
+			patch -s -p1 --forward -i $patch
 			echo -n -e "\e[32m $(basename $patch)\e[0m"
 		fi
 	done
